spec/jobs/pull_updates_spec.rb

```diff
@@ -20,6 +20,9 @@
 
   before do
     allow(HttpPuller).to receive(:new).and_return(http_puller)
+    allow(http_puller).to receive(:get_all).and_return('applications' => [])
+    allow(http_puller).to receive(:get_all).with(since: Time.zone.local(2023, 1, 1), count: 100)
```

Comment:
Nit: It's not at all clear from this spec why http_puller would receive a `since` of specifically `Time.zone.local(2023, 1, 1)`. The fallback to that date isn't something we mention in a different spec. I wonder if as a marginal improvement, PullUpdates should define that date as a constant and we could reference that constant here? But not crucial by any means.